src/api/common/connection.js

```diff
@@ -62,6 +73,10 @@ class Connection extends EventEmitter {
         }
         this.emit(data.id.toString(), data);
       } else if (isStreamMessageType(data.type)) {
+        if (data.type === 'ledgerClosed') {
```

Comment:
No, ripple-lib/core renames it from ledgerClosed to ledger_closed

https://ripple.com/build/rippled-apis/#subscribe
https://github.com/ripple/ripple-lib/blob/develop/src/core/remote.js#L503